core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1695,6 +1695,10 @@ public String toString() {
           .setDescription("How often the master schedules persistence jobs "
               + "for files written using ASYNC_THROUGH")
           .build();
+  public static final PropertyKey MASTER_PERSISTENCE_BLACKLIST =
+      new Builder(Name.MASTER_PERSISTENCE_BLACKLIST)
+          .setDescription("Patterns to blacklist persist, comma separated, string match, no regex.")
```

Comment:
does this apply to all persistence concepts, include CACHE_THROUGH, as well as a user CLI initiated persist rpc?

Also, is this string match for any part of the full path, or just the last component? Can the string match span directories, like `o/b` for `/foo/bar/`?

Reply:
It does not apply to cache through, it applies to cli `persist`. String match is on the full string.

Reply:
Updated the description